README.md

```diff
@@ -1,7 +1,8 @@
 tideline
 ========
 
-[![Build Status](https://travis-ci.org/tidepool-org/tideline.png?branch=master)](https://travis-ci.org/tidepool-org/tideline)
+[![Build Status](https://img.shields.io/travis/tidepool-org/tideline/master.svg)](https://travis-ci.org/tidepool-org/tideline)
```

Comment:
Switching the build status badge to use shields.io for consistent look and feel.